pages/docs/manual/latest/pattern-matching-destructuring.mdx

```diff
@@ -580,6 +580,35 @@ Here are some advices.
 
 Do not abuse the wildcard `_` too much. This prevents the compiler from giving you better exhaustiveness check, which would be especially important after a refactoring where you add a new case to a variant. Try only using `_` against infinite possibilities, e.g. string, int, etc.
 
+If you're about to use a wildcard when matching on a large but finite set of possibilities because you have many possibilities that return the same thing, consider reaching for disjunctive patterns instead. That is, you can replace:
```

Comment:
This sentence is very long. Can you divide it into shorter sentences?
Is it worth introducing the term `disjunctive patterns`?  What if we just talk about "writing out all cases explicitly"?

Reply:
I'll rewrite it to be more approachable. My only reason for using the term was to give readers a specific term that they can do more research on.

This brings up the larger idea of a glossary section which gives uncommon terms and brief definitions allowing people to grok concepts as well as giving them specific keywords with which to do independent research. This probably deserves an issue and longer discussion though.